Forms/CustomerForm.cs

```diff
@@ -0,0 +1,149 @@
+    {
+        private readonly IManager<Customer> _manager = new CustomerManager();
+
+        public CustomerForm()
+        {
+            InitializeComponent();
+        }
+
+        public void LoadData()
+        {
+            try
+            {
+                customerListView.Items.Clear();
+                List<ListViewItem> listViewItems = new List<ListViewItem>();
+
+                foreach (Customer customer in _manager.GetAll())
+                {
+                    listViewItems.Add(new ListViewItem(new[] {customer.Id.ToString(),
+                                                          $"{customer.FirstName} {customer.MiddleName} {customer.LastName}",
+                                                          customer.Address}));
+                }
+
+                customerListView.Items.AddRange(listViewItems.ToArray());
+            }
+            catch (Exception ex)
+            {
+                Logger.log.Error(ex.ToString());
+            }
+        }
+
+        public void EnableButtons (bool value)
+        {
+            confirmButton.Enabled = value;
+            signUpButton.Enabled = value;
+        }
+
+        private void ConfirmButton_Click(object sender, System.EventArgs e)
+        {
+            try
+            {
+                int id = textBoxMemberId.Text.ToInt(-1);
+                Customer customer = _manager.GetById(id);
+
+                if (customer == null)
+                {
+                    string message = "Entered ID can't be found.";
+                    string caption = "Please try again.";
+                    MessageBox.Show(message, caption, MessageBoxButtons.OK);
+                }
+                else
+                {
+                    EnableButtons(false);
+                    ProfileForm profileForm = new ProfileForm(id);
+                    profileForm.MdiParent = this.ParentForm;
+                    profileForm.Show();
+                }
+            }
+            catch (Exception ex)
+            {
+                Logger.log.Error(ex.ToString());
+            }
+        }
+
+        private void SignUpButton_Click(object sender, System.EventArgs e)
+        {
+            EnableButtons(false);
+            SignUpForm signUpForm = new SignUpForm();
+            signUpForm.MdiParent = this.ParentForm;
+            signUpForm.Show();
+        }
+
+        private void CustomerListView_Click(object sender, System.EventArgs e)
+        {
+            if (customerListView.SelectedItems.Count > 0)
+            {
+                textBoxMemberId.Text = customerListView.SelectedItems[0].Text;
+            }
+        }
+
+        private void SearchButton_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                string searchName = textBoxSearchName.Text;
+                string searchAddress = textBoxSearchAddress.Text;
+
+                if (!string.IsNullOrEmpty(searchName) || !string.IsNullOrEmpty(searchAddress))

```

Comment:
use string.IsNullOrWhitespace so that it can check if the string only contains whitespace